README.md

```diff
@@ -60,12 +60,18 @@ npm i --save-dev enzyme
 Enzyme is currently compatible with `React 15.x`, `React 0.14.x` and `React 0.13.x`. In order to
 achieve this compatibility, some dependencies cannot be explicitly listed in our `package.json`.
 
-If you are using `React 0.14` or `React 15.x`, in addition to `enzyme`, you will have to ensure that
+If you are using `React 0.14` or `React <15.5`, in addition to `enzyme`, you will have to ensure that
 you also have the following npm modules installed if they were not already:
 
 ```bash
-npm i --save-dev react-addons-test-utils
-npm i --save-dev react-dom
+npm i --save-dev react-addons-test-utils react-dom
+```
+
+If you are using `React >=15.5`, in addition to `enzyme`, you will have to ensure that you also have
+the following npm modules installed if they were not already:
+
+```bash
+npm i --save-dev react-dom create-react-class
```

Comment:
i'd agree, but most of our usage is in tests of createClass components, so the only option there would be to do away with those tests altogether 

(its use in  ReactWrapperComponent can be replaced with an ES6 class easily)